src/App.tsx

```diff
@@ -1,18 +1,91 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useContext, useMemo, useState } from 'react';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodosContext } from './components/context/TodoContext';
+import { Todo } from './types/Todo';
+import { Status } from './types/StatusEnum';
 
 export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [filterStatus, setFilterStatus] = useState(Status.All);
+
+  const todosContext = useContext(TodosContext);
+
+  if (!todosContext) {
+    return null;
+  }
+
+  const {
+    todos,
+    addTodo,
+    deleteCompletedTodos,
+  } = todosContext;
```

Comment:
it's not good way to check todosContext and return null
for what? in which scenario we will not have todosContext? in context directly I see you added all necessary values